014-quarkus-panache-with-transactions-xa/src/main/java/io/quarkus/qe/rest/data/UserRepository.java

```diff
@@ -2,17 +2,8 @@
 
 import javax.enterprise.context.ApplicationScoped;
 
-import io.quarkus.hibernate.orm.panache.PanacheQuery;
 import io.quarkus.hibernate.orm.panache.PanacheRepositoryBase;
-import io.quarkus.panache.common.Sort;
 
 @ApplicationScoped
 public class UserRepository implements PanacheRepositoryBase<UserEntity, Long> {
-
-    //This will override behaviour of HTTP GET method of PanacheResource
-    //TODO change to findAll() once feature is implemented https://github.com/quarkusio/quarkus/issues/16871
-    @Override
-    public PanacheQuery<UserEntity> findAll(Sort sort) {
-        return find("select u from user_entity u order by u.name asc");
-    }
```

Comment:
This method should remain but be changed to `public PanacheQuery<UserEntity> findAll()`.
The point was to override the default behavior of HTTP GET, without touching HTTP GET?sort=

As I recall it right, if you do `get("/users/all?sort=name")` and `get("/users/all?sort=!name")` the output was the same (because we overrided findAll(Sort sort) to sort ascending)

The fixed issue should verify that by overriding `findAll()` we do override HTTP GET **without** parameters
&& by overriding `findAll(Sort sort)` we override HTTP GET **with** parameters

Before, we could change the behavior of HTTP GET without parameters only by overriding `findAll(Sort sort)`, but this will also affect HTTP GET with parameters (which is undesirable)